fhir-bulkimportexport-webapp/src/main/java/com/ibm/fhir/bulkimport/ImportTransientUserData.java

```diff
@@ -18,6 +20,9 @@
     private ByteArrayOutputStream bufferStream4ImportError = new ByteArrayOutputStream();
```

Comment:
On line 14, add a space between ImportCheckPointData{   so it is `ImportCheckPointData {`

Reply:
Also, I'd suggest generating the serialVersionUID in this case.  1L may have unintended consequences

Reply:
yeah, good catch. somehow I always failed/fails to generate the it for this one. :(

Reply:
fixed via using command line instead of using eclipse.

Reply:
done